components/ws-daemon/pkg/cpulimit/cpulimit.go

```diff
@@ -222,79 +222,6 @@ func (f fixedLimiter) Limit(budgetSpent CPUTime) (newLimit Bandwidth) {
 	return f.FixedLimit
 }
 
-// Bucket describes a "pot of CPU time" which can be spent at a particular rate.
-type Bucket struct {
```

Comment:
This is proof that it has not been used because this part has been erased and the other parts have not been changed.